### DISCUSSION TITLE
how to connect vs code to git hub

### DISCUSSION THREAD

**Discussion by hjasaiwal114** [community/community#48182] [New to GitHub]
hi,  i want to push my project in github but when hi use git push -u origin master 
"please make sure you have correct acces rights"
its my persnol project and my ssh and http key is right please give me resource or solution 
thanks in advance

**Comment by Ntshangase** [comment]
Hello, Check out this documentation, I might be helpful https://code.visualstudio.com/docs/sourcecontrol/github

**Reply by hjasaiwal114** [reply]
thank you so much for the link through this article  i am able to know more about github

**Reply by Ntshangase** [reply]
If you found the answer helpful you can "mark as answer"

**Reply by Ntshangase** [reply]
@hjasaiwal114 were you able to solve your query?
